scripts/update_material_icon_names.py

```diff
@@ -17,6 +17,17 @@
 """Update the list of material icon names in `lib/streamlit/material_icon_names.py.
 
 Please only run this when updating the material symbols font to a later version.
+
+To update the material symbols font file, follow these steps:
```

Comment:
We could think about extend the script to automatically download the latest font and replace it with the existing one as part of the update script. I added a tech debt note [here](https://www.notion.so/snowflake-corp/Tech-Debt-DB-8613db6852c84a039c4e1c8e929507c4?p=1387170bb416802bbd41c6789fdbbc29&pm=s).